pkgs/applications/misc/makeself/default.nix

```diff
@@ -11,7 +11,9 @@ stdenv.mkDerivation rec {
     sha256 = "1lw3gx1zpzp2wmzrw5v7k31vfsrdzadqha9ni309fp07g8inrr9n";
   };
 
-  patchPhase = ''
+  patches = [ ./Use-rm-from-PATH.patch ];
```

Comment:
Please switch to `fetchpatch` since upstream is now convinced.

Reply:
The commit from my PR, which is now merged upstream, doesn't apply as a patch to 2.4.0, unfortunately.

Reply:
Ok. Please add:

```
# backported from https://github.com/megastep/makeself/commit/77156e28ff21231c400423facc7049d9c60fd1bd
```

comment so know we can remove this later.

Reply:
Done.